zstacklib/ansible/zstacklib.py

```
@@ -2651,6 +2651,12 @@ def get_qemu_img_version(host_post_info):
     return status, qemu_img_version
 
 
+def get_libcbd_version():
+    command = "qemu-img --version | grep 'qemu-img version' | cut -d ' ' -f 3 | cut -d '(' -f 1"
+    (status, qemu_img_version) = run_remote_command(command, host_post_info, False, True)
```

Ruff (0.14.5)

Undefined name `host_post_info`

(F821)